sdk/search/tests.yml

```diff
@@ -10,6 +10,7 @@ jobs:
       MaxParallel: 3
       EnvVars:
         AZURE_TEST_MODE: LIVE
+        AZURE_RESOURCEGROUP_NAME: ${AZURE_RESOURCEGROUP_NAME}
```

Comment:
This isn't needed, this is only required when using secrets retrieved from KeyVault or the environment configuration doesn't match what is expected in the tests.